README.md

```diff
@@ -20,17 +20,17 @@ Note: [Homebrew](https://brew.sh/) is required for Mac devices.
         - Ubuntu: `sudo apt install mecab mecab-ipadic-utf8`
         - Fedora: `sudo dnf install mecab mecab-ipadic`
         - Arch: `sudo pacman -S mecab-git mecab-ipadic`
-3. [Download](https://github.com/themoeway/yomitan-mecab-installer/archive/master.zip) and extract this repository to the location you wish to install the files.
+3. [Download](https://github.com/yomidevs/yomitan-mecab-installer/archive/master.zip) and extract this repository to the location you wish to install the files.
```

Comment:
wouldnt it be easier to make all these paths local references

Reply:
The rest of them can be but not the specific one you highlighted there since that's a link to download the repo itself. Will change the others.